.github/workflows/ci.yml

```diff
@@ -18,23 +18,26 @@ jobs:
         configuration: [Debug, Release]
         include:
           - os: windows-latest
-            cmake-parameters: -G "Visual Studio 16 2019" -A x64
+            cmake-generator: -G "Visual Studio 16 2019" -A x64
+            cmake-build-param: -j $env:NUMBER_OF_PROCESSORS
             folder: win64
           - os: ubuntu-latest
-            cmake-parameters: -G "Unix Makefiles"
+            cmake-generator: -G "Unix Makefiles"
+            cmake-build-param: ${{ null }}
             folder: linux64
           - os: macos-latest
-            cmake-parameters: -G Xcode
+            cmake-generator: -G "Xcode"
+            cmake-build-param: ${{ null }}
             folder: macos64
     steps:
       - uses: actions/checkout@v2
       - name: Generate CMake Files
         run: |
           mkdir build && cd build
-          cmake .. ${{ matrix.cmake-parameters }}
+          cmake .. ${{ matrix.cmake-generator }}
       - name: Build
         working-directory: build
-        run: cmake --build . --config ${{ matrix.configuration }}
+        run: cmake --build . --config ${{ matrix.configuration }} ${{ matrix.cmake-build-param }}
```

Comment:
With cmake-build-param option, msvc can build faster without need to make separate step.